sourcemash/frontend/templates/splash.tpl

```diff
@@ -0,0 +1,61 @@
+<div class="section no-pad-bot" id="splash-banner">
+  <div class="container">
+    <div class="row center">
+    <div class="brand-logo">
+      <img src="static/img/logotext.svg" alt="Sourcemash">
+    </div>
+    <h4 class="header col s12 light center">Read Your News. Faster.</h4>
+    </div>
+    <div class="row center">
+      <div class="btn btn-large waves-effect waves-light" id="need-account">Sign Up!</div>
+    </div>
+    <br>
+  </div>
+  <div id="splash-divider"></div>
+</div>
+<div class="section no-pad-bot" id="splash-info">
+  <div class="container">
+    <div class="row">
+
+      <h3 class="col s12 light center header flow-text" id="splash-subtitle">Sourcemash shortens the time it takes to read your daily news.</h3>
+
+    </div>
+
+    <div class="row">
+      <div class="col s4">
+        <div class="center promo">
+          <i class="mdi-editor-insert-comment promo-icon"></i>
+        </div>
+      </div>
+      <div class="col s8">
+        <p class="promo-caption">Choose your sources</p>
+        <p class="light">We want you to read the sources that you want to read. Sourcemash has a growing number of RSS feeds ready to browse, but if you don't see what you're looking for, add it! Read your news, not somebody else's version.</p>
```

Comment:
"Read the sources you want to read. Select from our list of feeds, or add your own!. We categorize your news, not someone else's."